src/PeriodEnd/TransferPayments/SFA.DAS.ProviderPayments.Calc.TransferPayments/Dal/Repositories/AccountRepository.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using SFA.DAS.Payments.DCFS.Infrastructure.Data;
+using SFA.DAS.ProviderPayments.Calc.TransferPayments.Dal.DatabaseEntities;
+using SFA.DAS.ProviderPayments.Calc.TransferPayments.Dependencies;
+using SFA.DAS.ProviderPayments.Calc.TransferPayments.Domain;
+
+namespace SFA.DAS.ProviderPayments.Calc.TransferPayments.Dal.Repositories
+{
+    class AccountRepository : DcfsRepository, IAmAnAccountRepository
+    {
+        private static ConcurrentDictionary<long, Account> _accounts;
```

Comment:
does this need to be static? If it does, it should be renamed accounts. If not, change it